sympy/combinatorics/pc_groups.py

```diff
@@ -106,17 +110,18 @@ def minimal_uncollected_subwords(self, word):
             s2, e2 = array[i+1]
             s = ((s1, 1), )
             s = group.dtype(s)
-            if e1 > 0 and e1 < re[s]:
-                if e2 > 0 and index[s1] > index[s2]:
-                    # case-0:  v = x[i]**a*x[i+1], where index[x[i]] > index[x[i+1]]
-                    uncollected_subwords[((s1, e1), (s2, 1))] = 0
-
-                elif e2 < 0 and index[s1] > index[s2]:
-                    # case-1: v = x[i]**a*x[i+1]*-1, where index[x[i]] > index[x[i+1]]
-                    uncollected_subwords[((s1, e1), (s2, -1))] = 1
             if e1 > re[s]-1:
```

Comment:
What about `e1 < 0`?

Reply:
There is a `continue` in this `if e1 > re[s] -1` so, in all other cases the below `e2` section of code will work!

Reply:
> What about `e1 < 0`?

I doubt that `e1 < 0` will be uncollected because further this will be mapped to power relation and we can't have `negative relative order`!

Reply:
It is uncollected if the relative order is finite (Definition 8.14 a). Someone may try to collect words with negative exponents.

Reply:
But, that will lead to error because power-relators are in form of positive exponent and have a look at example 8.7 if negative-exponents should be considered then that example should have been proceeded further.

Reply:
See the lines 7 - 9 in COLLECTED_WORD on pg. 282.

Reply:
In example 8.7, the relative order of x2 is infinite.

Reply:
Ahh! that's right!